common/Jet_QA.C

```diff
@@ -344,22 +348,43 @@ R__LOAD_LIBRARY(libjetqa.so)
 	// --------------------------------------------------------------------------
 	inline std::vector<uint32_t> GetJetsToQA()
 	{
-
-		std::vector<uint32_t> vecJetsToQA = {
+	     std::vector<uint32_t> vecJetsToQA;
+	     if (HasCalos)
+	     {
+		   if (QA_SubtractedTowers)
```

Comment:
In line with above suggestion:
```suggestion
		   if (UseBkgdSub)
```